src/App.vue

```diff
@@ -61,8 +63,20 @@
       },
       setServiceLoaded() {
         // @ts-expect-error ts-migrate(2339) FIXME: Property 'api' does not exist on type 'Window & typeof globalThis'
-        window.api?.response("sendServiceStatus", ({ id, output }) => {
+        window.api?.response("sendServiceStatus", async ({ id, output }) => {
           console.log(id, output, "output")
+          if (output == "120") {
+            const experienceObj = settingsModule.data.listOfMenu.filter(
+              (element) => element.service == id
+            )
+            if (experienceObj.length > 0) {
+              experienceObj[0].disabled = false
+              experienceObj[0].icon =
+                settingsModule.data.previousIcon || experienceObj[0].icon
+              settingsModule.data.previousIcon = ""
```

Comment:
what is the previousIcon refers to ?
You can have two things activeIcon (icon) and experienceIcon (actual icon for the experience)

That should along all the objects of the listOfMenu (List of experience)

Reply:
Done